Task1_UserAPI/MeetingManager.Tests/UserCrudTest.cs

```diff
@@ -0,0 +1,160 @@
+﻿using AutoMapper;
+using MeetingManager.Core.Entities;
+using MeetingManager.Core.Interfaces;
+using MeetingManager.Core.Models;
+using MeetingManager.Core.Services;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MeetingManager.Tests
+{
+    [TestClass]
+    public class UserCrudTest
+    {
+        private IMapper mapper;
+
+        private List<User> users;
+
+        private IUserRepository mockUserRepository;
+
+        private IUserService userService;
+
+        public UserCrudTest()
+        {
+            //TestData
+            users = new List<User>()
+            {
+                new User { Id = 1, Email = "Test1@mail.com", FirstName = "Test1", LastName = "Test1", Password = "HashPassword1"},
+                new User { Id = 2, Email = "Test2@mail.com", FirstName = "Test2", LastName = "Test2", Password = "HashPassword2"},
+                new User { Id = 3, Email = "Test3@mail.com", FirstName = "Test3", LastName = "Test3", Password = "HashPassword3"},
+                new User { Id = 4, Email = "Test4@mail.com", FirstName = "Test4", LastName = "Test4", Password = "HashPassword4"}
+            };
+
+            //MockRepo
+            Mock<IUserRepository> mockRepository = new Mock<IUserRepository>();
+            SetupMock(mockRepository);
+            mockUserRepository = mockRepository.Object;
+
+            //Mapper
+            var config = new MapperConfiguration(cfg => {
+                cfg.CreateMap<User, UserModel>();
+                cfg.CreateMap<UserModel, User>();
+            });
+            mapper = new Mapper(config);
+
+            //Service
+            userService = new UserService(mockUserRepository, mapper);
+        }
+
+        [TestMethod]
+        public async Task CanReturnAllUsers()
```

Comment:
the name of the test  methods should describe their action
CanReturnAllUsers() - GetAllUsers_ReturnsUsers